include/ruby/internal/abi.h

```diff
@@ -24,7 +24,7 @@
  * In released versions of Ruby, this number is not defined since teeny
  * versions of Ruby should guarantee ABI compatibility.
  */
-#define RUBY_ABI_VERSION 1
+#define RUBY_ABI_VERSION 2
```

Comment:
What needs this?

Reply:
`include/ruby/internal/gc.h` changed in an incompatible way so we need to bump the ABI version.